mail_activity_cancel_tracking/tests/test_mail_activity_cancel_tracking.py

```diff
@@ -32,3 +33,8 @@ def test_mail_activity_cancel(self):
             "mail_activity_cancel_tracking_cancel",
             login="test-user",
         )
+
+    @mute_logger("odoo.models.unlink")
+    def test_record_unlink(self):
+        self.partner.unlink()
+        self.assertFalse(self.partner.exists())
```

Comment:
This is not adding value. Just put a comment on the line above saying that this shouldn't fail.